src/main/java/net/pms/network/RequestV2.java

```diff
@@ -618,89 +618,30 @@ public ChannelFuture answer(
 				}
 			}
 		} else if ((method.equals("GET") || method.equals("HEAD")) && (argument.toLowerCase().endsWith(".png") || argument.toLowerCase().endsWith(".jpg") || argument.toLowerCase().endsWith(".jpeg"))) {
-			if (argument.toLowerCase().endsWith(".png")) {
-				output.headers().set(HttpHeaders.Names.CONTENT_TYPE, "image/png");
-			} else {
-				output.headers().set(HttpHeaders.Names.CONTENT_TYPE, "image/jpeg");
-			}
-
-			output.headers().set(HttpHeaders.Names.ACCEPT_RANGES, HttpHeaders.Values.BYTES);
-			output.headers().set(HttpHeaders.Names.CONNECTION, HttpHeaders.Values.KEEP_ALIVE);
-			output.headers().set(HttpHeaders.Names.EXPIRES, getFUTUREDATE() + " GMT");
-			inputStream = getResourceInputStream(argument);
+			inputStream = imageHandler(output);
 		} else if ((method.equals("GET") || method.equals("HEAD")) && (argument.equals("description/fetch") || argument.endsWith("1.0.xml"))) {
 			output.headers().set(HttpHeaders.Names.CONTENT_TYPE, "text/xml; charset=\"utf-8\"");
```

Comment:
@drakulis  I'm sure there is a good reason for this but why are these lines being left out of the new methods?

Reply:
@SubJunk Yes, as you see we have nice pattern here. Take a look, I think 80% of requests are setting this header this way. It means that `CONTENT_TYPE` should be by default set to `"text/xml; charset=\"utf-8\"` and rest of requests should have own specific `CONTENT_TYPE`. I want to clean it up in some next batch of changes

Reply:
Cool that sounds good